protocol/0073-LIMN-limited_network_life.md

```diff
@@ -260,7 +261,7 @@ If for `network.checkpoint.timeElapsedBetweenCheckpoints` the value is set to `0
 
 #### Test case 6.2: `timeElapsedBetweenCheckpoints` set to value outside acceptable range
 
-### Test case 11: Rewards are distributed correctly every epoch including with the use of recurring transfers (<a name="0073-LIMN-022" href="#0073-LIMN-022">0073-LIMN-022</a>)
+### Test case 11: Rewards are distributed correctly every epoch including with the use of recurring transfers (<a name="0073-LIMN-022" href="#0073-LIMN-022">0073-LIMN-022</a>)(<a name="0073-SP-LIMN-022" href="#0073-SP-LIMN-022">0073-SP-LIMN-022</a>)
```

Comment:
I don't think we need this new spot AC. Recurring transfers to the general reward pool should be unaffected by adding spots.